src/extension/background.js

```diff
@@ -576,6 +578,23 @@ const internalActions = {
     chrome.contextMenus.onClicked.addListener(async ({ menuItemId }, tab) => {
       if (!tab.url) return;
       internalActions[menuItemId](tab);
+      await chrome.scripting.executeScript({
+        target: { tabId: tab.id },
+        func: async () => {
+          try {
+            const sampleRUM = await import(chrome.runtime.getURL('rum.js'));
```

Comment:
`import()` returns the module, not the default export (see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import) for details). so you will need to do something like this to get a reference to the `sampleRUM` function:
```suggestion
            const mod = await import(chrome.runtime.getURL('rum.js'));
            const { default: sampleRUM } = mod;
```